pages/_document.tsx

```diff
@@ -3,7 +3,12 @@ import { Html, Head, Main, NextScript } from "next/document";
 export default function Document() {
   return (
     <Html lang="fi">
-      <Head />
+      <Head>
+        <link
+          rel="shortcut icon"
+          href="https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/120/twitter/322/rainbow-flag_1f3f3-fe0f-200d-1f308.png"
+        />
```

Comment:
1) Didn't expect this to be how one favicons here! 🆒 
2) Could you also remove the` public/favicon.ico` file that is a leftover from the boilerplate in this commit?

Reply:
Yes, good catch! I will remove it.